src/batou_ext/archive.py

```diff
@@ -15,6 +19,11 @@ class SingleUntar(batou.lib.archive.Untar):
 
     _required_params_ = {"target": "."}
 
+    cleanup_markers_after_days = Attribute(int, default=None)
+
+    _markers_to_unlink = []
```

Comment:
Mutable class variables are usually a bad idea unless to want to share state over all instances.

Reply:
Eh, you're right. Variables are now set in configure().